### PR TITLE
Duo Tailsitter Configuration guide Documentation

### PR DESCRIPTION
**Pull request by HarshilNaikk**
As discussed with @sfuhrer I've added an initial draft of the duo-tailsitter configuration and tuning documentation. This is not the final draft ofcourse. I would love reviews from the maintainers as to what can and should be added/removed. I've also secured permission from my company to add pictures and videos of testing, which I will in a follow up commit. But other than that, let me know what to edit contentwise. 

Thanks a lot! Happy to contribute!

### CODE REVIEW

**Comment by hamishwillee** [comment]
Hi @HarshilNaikk 

There is something wrong with your git setup. If you look at the files that are affected, you've deleted a whole bunch of files in the `/en/msg_docs` folder. You  have also deleted files in the translation folders - i.e. folders that don't start with /en/ and should never be touched.

If I had to fix this I would copy the vtol file you meant to touch into another directory. Then in your fork of the docs I would start from scratch, then copy your modified file over. In git command line, where upstream is this repo, and origin is your repo it would look like

```
# manually copy the file somewhere

# Get upstream main - the unchanged form
git fetch upstream main
#reset your branch to upstream/main
git reset --hard upstream/main
# Do git status to check that your tree is fine and there are no orphan files floating around. 
git status
# If it is, copy the original file you changed back in place into the /en tree.

# Then submit the changes
git add .
git commit -m "My new vtol tuning"

# then force push the branch here again
git push origin <your branch name> --force
```

Then we can review it :-) I'm out for a week but I am sure the others will give you technical review in the meantime. Thanks so much for contributing.

**Comment by HarshilNaikk** [comment]
Whoops. Need to fix that.

File: en/config_vtol/vtol_tailsitter_tuning.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+The MC tuning is tricky, as certain axes depend on the control surfaces, and certain ones depend on the motors.
+
+:::warning
+Always disable [MC_AIRMODE](../advanced_config/parameter_reference.md#MC_AIRMODE) when tuning a vehicle.
+:::
+
+To tune the tailsitter in MC mode, we start off by setting some gains to 0. Just as a sanity check. 
+- Roll rate control ([MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) = 0, [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I) = 0, [MC_ROLLRATE_D](../advanced_config/parameter_reference.md#MC_ROLLRATE_D) = 0, [MC_ROLLRATE_K](../advanced_config/parameter_reference.md#MC_ROLLRATE_K) = 1)
+- Pitch rate control ([MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P) = 0, [MC_PITCHRATE_I](../advanced_config/parameter_reference.md#MC_PITCHRATE_I) = 0, [MC_PITCHRATE_D](../advanced_config/parameter_reference.md#MC_PITCHRATE_D) = 0, [MC_PITCHRATE_K](../advanced_config/parameter_reference.md#MC_PITCHRATE_K) = 1)
```
Comment (sfuhrer) [inline en/config_vtol/vtol_tailsitter_tuning.md:39]: Actually MC_PITCHRATE_X and MC_YAWRATE_X params shouldn't have an effect on the tuning of a duo tailsitter. In hover mode it uses the MC controller for the axes actuated by motors (only roll), while it uses the fixed-wing controller for pitch and yaw. So you normally tune:

For hover yaw:
FW_RR_P
FW_RR_I
FW_RR_FF (I recommend to keep this low, or even at 0)
and for hover pitch:
FW_PR_P
FW_P_I
FW_PR_FF

The convention is that control surfaces get controlled through the FW controller, and that for the FW controller the body axes are named as they are oriented in fixed-wing flight. 
I acknowledge this is insanely confusing, and thus appreciate it even more that you started with this tuning guide!

File: en/config_vtol/vtol_tailsitter_tuning.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+### Angle Gains
+
+Once you have a tuned rate controller, your vehicle can hover in a very stable way (albeit with some support for position control, since your angle gains are still 0 and your RC transmitter will not work).
+
+You should then move on to tuning the Angle gains. In most cases, just a Proportional controller is sufficient enough for this. 
+
+- Repeat the same process as mentioned above in the Rate controller gain tuning, but instead of manually displacing the vehicle in a particular axes, use the RC transmitter to give pulses in certain axes and check the responce. Tune the Proportional gains so that there are no oscillations, and if they do exist, ensure that they subside quickly.
+
+
+## Duo Tailsitter FW Mode Tuning Guide
```
Comment (sfuhrer) [inline en/config_vtol/vtol_tailsitter_tuning.md:64]: Do you fly with or without an airspeed sensor? As that changes how you should approach the tuning. With an airspeed sensor the airspeed scaling should take care of the gain scheduling automatically, while if you fly without airspeed sensor you need to tune it through the setting of FW_AIRSPD_STALL and FW_AIRSPD_TRIM. The bigger those two numbers are apart, the more gain scheduling happens.

**Review by sfuhrer — COMMENTED** [review]
Thanks a lot for kicking this off! Tailsitter tuning is super complex, I hope first of all to be able to guide you through it and then to create a step by step guide here that enables other developers as well.

**Comment by hamishwillee** [comment]
@sfuhrer This was closed in favour of #2593

I cross posted your comments.

**Pull request closed without merging**